src/ORM/Query.php

```diff
@@ -1181,22 +1203,51 @@ public function jsonSerialize()
     }
 
     /**
+     * Sets whether or not the ORM should automatically append fields.
+     *
+     * By default calling select() will disable auto-fields. You can re-enable
+     * auto-fields with this method.
+     *
+     * @param bool $value Set true to enable, false to disable.
+     * @return self
+     */
+    public function enableAutoFields($value)
+    {
+        $this->_autoFields = (bool)$value;
```

Comment:
Is this the common way to deal with boolean type protected/private members? Or should it either do nothing or check is_bool()?